src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -590,6 +590,111 @@ static void insertConvTransposeSpatialDim(SmallVectorImpl<int64_t> &outputDims,
   }
 }
 
+//===----------------------------------------------------------------------===//
+// Support function to emit a warning if an operator attibute value is greater
+// than a preset limit.
+//===----------------------------------------------------------------------===//
+
+static InFlightDiagnostic verifyAttributeLimit(
+    Operation &op, const Twine &attrName, int64_t attrVal) {
+  // Global limit for all attributes.
+  constexpr int64_t attrValLimit = 100;
+
+  if (abs(attrVal) > attrValLimit) {
+    Twine msg(op.getName().getStringRef());
+    std::string limitStr =
+        std::to_string((attrVal > 0) ? attrValLimit : -attrValLimit);
+
+    return emitWarning(op.getLoc(),
```

Comment:
Does the warning return a failure? Maybe a quick comment may help here clarify this.